engines/dfc_provider/dfc_provider.gemspec

```diff
@@ -0,0 +1,22 @@
+# frozen_string_literal: true
+
+$LOAD_PATH.push File.expand_path('lib', __dir__)
+
+# Maintain your gem's version:
+require "dfc_provider/version"
+
+# Describe your gem and declare its dependencies:
+Gem::Specification.new do |spec|
+  spec.name        = 'dfc_provider'
+  spec.version     = DfcProvider::VERSION
+  spec.authors     = ['Admin OFF']
+  spec.email       = ['admin@openfoodfrance.org']
```

Comment:
I wouldnt add OFF specific contact here, I'd make this generic like in the other engines:
  s.authors     = ["developers@ofn"]